pages/index.js

```diff
@@ -1,11 +1,15 @@
+import dynamic from 'next/dynamic'
 import Head from 'next/head'
 import Meta from '@hackclub/meta'
 import Nav from '../components/nav'
 import Banner from '../components/recap/banner'
 import Scrapbook from '../components/recap/scrapbook'
 import HardwareGrants from '../components/recap/hardwaregrants'
 import Polaroid from '../components/recap/polaroid'
-import KawasakiDetector from '../components/recap/kawasakiDetector'
+const KawasakiDetector = dynamic(
+  () => import('../components/recap/kawasakiDetector'),
+  { loading: () => <p>...</p> }
+)
 import MIDI from '../components/recap/midi'
 import LedMusic from '../components/recap/ledMusic'
```

Comment:
This isn’t a big deal, but the fact that you mix `formattingLike`, `thisand`, `like-this` across your components is :)

Reply:
... unique

Reply:
on the other side they are all quite nice tho, im just saying